plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/StatsMetricPublisher.java

```diff
@@ -20,18 +23,67 @@ public class StatsMetricPublisher {
 
     private final Stats stats = new Stats();
 
+    private final Map<BlobStore.Metric, Stats> extendedStats = new HashMap<>() {
+        {
+            put(BlobStore.Metric.REQUEST_LATENCY, new Stats());
```

Comment:
Do you think put and get can also be added here separately instead of one umbrella for all the requests ?
I assume request latency/success will have all type of requests on repository.

Reply:
Sorry, I am unable to follow you

Reply:
put and get are already added separately . Resolving this. @amkhar  : Pls reopen if i misunderstood.